src/FunderMaps.WebApi/Controllers/Report/IncidentController.cs

```diff
@@ -91,41 +96,24 @@ public async Task<IActionResult> GetAllAsync([FromQuery] PaginationModel paginat
         /// <summary>
         ///     Post a new incident to the backend.
         /// </summary>
-        /// <remarks>
-        ///     This call can be made anonymous or as a user.
-        /// </remarks>
         /// <param name="input"><see cref="IncidentDto"/></param>
         /// <returns><see cref="OkObjectResult"/></returns>
-        [AllowAnonymous]
         [HttpPost]
         [ProducesResponseType((int) HttpStatusCode.OK)]
         public async Task<IActionResult> CreateAsync([FromBody] IncidentDto input)
         {
             // Map.
             var incident = _mapper.Map<Incident>(input);
 
-            // Process the anonymity of the request.
-            if (_authManager.IsSignedIn(User))
-            {
-                var sessionUser = await _authManager.GetUserAsync(User);
-                var sessionOrganization = await _authManager.GetOrganizationAsync(User);
-
-                incident.Meta = new
-                {
-                    SessionUser = sessionUser.Id,
-                    sessionOrganization = sessionOrganization.Id,
-                    Gateway = Constants.IncidentGateway,
-                };
-            }
-            else
+            var sessionUser = await _authManager.GetUserAsync(User);
+            var sessionOrganization = await _authManager.GetOrganizationAsync(User);
+
+            incident.Meta = new
             {
-                incident.Meta = new
-                {
-                    UserAgent = Request.Headers["User-Agent"].ToString(),
-                    RemoteAddress = HttpContext.Connection.RemoteIpAddress?.ToString(),
-                    Gateway = Constants.IncidentGateway,
-                };
-            }
+                SessionUser = sessionUser.Id,
+                sessionOrganization = sessionOrganization.Id,
```

Comment:
Should also be PascalCase for consistency

Reply:
Nope, because its a local var.
![image](https://user-images.githubusercontent.com/7712841/92756795-6d6cb300-f38d-11ea-97d9-877bc5aa3bc3.png)

Reply:
I mean it like `SessionOrganization = sessionOrganization.Id,`

Reply:
I only now see what you mean. There is a missing capital. ofc. Fixing it